lib/active_merchant/billing/gateways/nuvei.rb

```diff
@@ -178,7 +178,7 @@ def scrub(transcript)
       private
 
       def network_transaction_id_from(response)
-        response.dig('paymentOption', 'paymentAccountReference')
+        response.dig('transactionId')
```

Comment:
This looks like the `transactionId` is just a generic id for the transaction, but can also be the NTID for a COF transaction?

Reply:
Hi @naashton yes, according to the Nuvei documentation the proper value is `transactionId`, this always represents the previous or original transaction CIT and should be sent as the `relatedTransactionId`. The  `paymentAccountReference ` does not represent a transaction id, it represents a unique identifier associated with a specific cardholder PAN.